modules/nw-egress-router-redirect-mode-ovn.adoc

```diff
@@ -30,46 +28,77 @@ metadata:
   name: egress-1
 spec:
   ports:
-  - name: database
+  - name: web-app
     protocol: TCP
-    port: 3306
+    port: 8080
   type: ClusterIP
   selector:
-    name: egress-router-pod
+    app: egress-router-cni
```

Comment:
@danielmellado, please confirm or correct me on the correct way to associate the service with the router.  (I need to state the label that is automatically assigned clearly in this section or another.)

Reply:
Yeah, you can check an example of this in https://etherpad.opendev.org/p/egress-router-cni-debug